src/computeScreenCoords.py

```diff
@@ -1,6 +1,70 @@
-def computeScreenCoords(eyeCoords):
+from enum import Enum
+import numpy as np
+from sklearn.linear_model import LinearRegression
+import pandas as pd 
+from scipy.interpolate import LinearNDInterpolator
+from typing import List, Tuple
+
+class InterpolationType(Enum):
+    LINEAR = 1
+    RBF_LINEAR = 2
+
+def unpackEyeCoords(eyeCoords: List[List[Tuple]]) -> List[Tuple]:
+    # unpacks [[(1,2),(3,4)], [(5,6), (7,8)]] to [(1,2,3,4), (5,6,7,8)]
+    return [sum(tupList,()) for tupList in eyeCoords]
+
+def unpackScreenCoords(screenCoords):
+    screenXCoords, screenYCoords = zip(*screenCoords)
+    return (list(screenXCoords), list(screenYCoords))
+
+class LinearRegression():
```

Comment:
It looks like you have both sklearn linear regression and scipy linear interpolation. Is the plan to offer both of these as options the user can select, or are they equivalent enough to only use one?

I think either one would work pretty well.

Reply:
Yeah, I did both as of now just to have more options for the user. As I add more I can either remove this or keep it if we see decent enough results in testing.